### DISCUSSION TITLE
Why I can't yank from vscode.neovim to windows system clipboard?

### DISCUSSION THREAD

**Discussion by zwyyy456** [vscode-neovim/vscode-neovim#1953] [Q&A] [answered]
The `init.lua` is in `~\AppData\Local\nvim`, and the content is as follow:

```lua
if vim.g.vscode then
  -- if true then
  -- Vscoe extension
  require("vscode.config.options")
  require("vscode.config.keymaps")
  require("vscode.config.lazy")
else
  -- bootstrap lazy.nvim, LazyVim and your plugins
  require("config.lazy")
  -- require("config.osc52")
end
```

while the content of `~\AppData\Local\nvim\lua\vscode\config\options.lua` is as follow:

```lua
-- clipboard
local opt = vim.opt
opt.clipboard = "unnamedplus" -- Sync with system clipboard
```
The version of vscode is `1.89.1` and neovim is `0.9.5`，and the version of vscode.neovim is `1.11.3`, in the past, I can yank from vscode.neovim to windows system clipboard, but today I find it failed.

Can anyone tell me how to solve this problem?

**Comment by xiyaowong** [comment]
Is it just an accident, or has it always failed?

**Comment by zwyyy456** [comment]
It fail always. How to find the reason? I try to run `:checkhealth` in vscode, the result is as follows:
![image](https://github.com/vscode-neovim/vscode-neovim/assets/45388797/be1f21ac-313e-42fe-8acf-352c04d2212c)

**Comment by zwyyy456** [comment, marked as answer]
I find the reason! I enabled `neovim clean` when I want to solve the problem of shaking while inputing Chinese.
